expo/gbe_forms_text.py

```diff
@@ -637,3 +635,61 @@ class time over threee days.  It is the original Professional Development Confer
     'title': "Note, this will change the title for all bookings of this event",
     'duration': "Enter duration as HH:MM:SS"
 }
+
+costume_proposal_form_text = '''<p>Thanks for your interest in our costume \
+    display. Each year a team of creative volunteers puts together a \
+    remarkable collection of burlesque costumes in our Exhibit Hall. The core \
+    of the team are BettySioux Tailor and Blitzen vonSchtupp, with the \
+    curatorial assistance of Miss Mina Murray.</p>\
+    <p>Costumes are selected for their historical significance, their 'wow' \
+    factor, their individuality (that is, we're not likely to take two \
+    similar costumes or more than one costume from the same person), and \
+    their craftsmanship. Not all costumes that are offered are accepted. \
+    You can check on the status of your submission on your personal \
+    homepage.</p>\
+    <p>If you are offering a costume for display, it is your responsibility \
+    to make sure that the costume is on-site before 9:00am on Saturday
+    morning. Dropping it off on Friday is fine. If you drop it off before \
+    6:00pm on Friday, it is likely it will be on display during The \
+    Bordello. Costumes will be available for pick-up after 9:00pm on \
+    Sunday evening. It is your responsibility to pick up your costume \
+    and make sure all pieces are retrieved when you do so. In rare cases \
+    and with previous arrangements, a costume can be picked up earlier on \
+    Sunday or shipped to you after The Expo.</p>\
+    <p>If you have questions about The Costume Display, please drop a \
+    note to BettySioux Tailor at \
+    <a href="mailto:Costumes@Burlesque-Expo.com">\
+    Costumes@Burlesque-Expo.com</a>. If you'd like to be one of the \
+    volunteers who puts up the display or one of the docents who helps \
+    educate people about the costumes over the weekend, please check our \
+    Volunteer Opportunities. '''
+
+costume_proposal_help_texts = {
+    'title': '''Please put the stage name, legal name, or company name of \
```

Comment:
Mostly out of curiosity, why not? That's not a challenge, it's a request 
for knowledge so I can understand better, and possibly avoid creating 
situations in the future where the business needs conflict with the 
technical needs.